recipe/meta.yaml

```diff
@@ -24,33 +24,29 @@ requirements:
     - wheel
   run:
     - python
-    - jinja2 >=2.10.1
     # we need to bound this since 198 changed some of the api
     - boto3 >=1.9.201
     - botocore >=1.12.201
     - cryptography >=3.3.1
-    - dataclasses  # [py<37]
-    - ecdsa !=0.15
     - requests >=2.5
     - xmltodict
     - werkzeug >=0.5,!=2.2.0,!=2.2.1
-    - pyyaml >=5.1
-    - pytz
     - python-dateutil >=2.1,<3.0.0
-    - python-jose >=3.1.0,<4.0.0
-    - docker-py >=2.5.1
     - responses >=0.13.0
-    - MarkupSafe !=2.0.0a1
-    - cfn-lint >=0.4.0
-    - flask
+    - jinja2 >=2.10.1
+    - importlib_metadata  # [py<38]
+    - python-jose[cryptography] >=3.1.0,<4.0.0
+    - ecdsa !=0.15
+    - docker-py >=2.5.1
+    - graphql-core
+    - pyyaml >=5.1
+    - cfn-lint >=0.40.0
+    - sshpubkeys >=3.1.0
+    - flask !=2.2.0,!=2.2.1
     - flask_cors
     - jsondiff >=1.1.2
```

Comment:
`pyparsing>=3.0.7` is missing, see https://github.com/getmoto/moto/blob/642ef7841cf7833d26bd21dd5eaab064b19eee69/setup.cfg#LL53C5-L53C21